src/lib/getContextPage.ts

```diff
@@ -53,7 +53,7 @@ export function getContextPageFromClient(): {
 
 export const formatOwnerTypes = (path: string) => {
   const type = path.split("/")[1]
-  let formattedType = camelCase(type)
+  let formattedType = camelCase(type).replace("2", "")
```

Comment:
Can we add a comment here describing what this does? This change could be unclear in the future